css-two-column-layout/index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html>
+<head>
+ <title>CSS Two Column Layout</title>
+ <link rel="stylesheet" type="text/css" href="styles.css">
+
+</head>
+<body>
+  <div class="container">
+    <h1>SCALE, SPEED, AGILITY</h1>
+    <p class="caption">We've dedicated thousands of man hours to making healthcare
+      institutions in the Helium<br> network run 10x more efficiently
+      and deliver an unparalleled quality of care.
+    </p>
+
+    <br>
+    <br>
+    <br>
+
+    <div class="row">
+      <div class="column">
+        <div class="row1">
+          <div class="column1">
```

Comment:
You shouldn't make numbered versions of classes like this. You should reuse the original class, and make an additional class you can add to make any additional styling changes you want to use. So for instance, instead of `row`, Use `row` and make another class for any styling you need in addition to the `display: flex` already applied to the `row` class.